pallets/runtime/tests/src/pips_test.rs

```diff
@@ -73,9 +73,11 @@ macro_rules! assert_no_pip {
 }
 
 fn make_proposal(value: u64) -> Call {
-//    Call::Pips(pips::Call::set_min_proposal_deposit(value.into()))
-    let content = vec![b'X'; 100];
-    Call::System(frame_system::Call::remark(content)).into()
+    Call::Pips(pips::Call::set_min_proposal_deposit(value.into()))
+}
+
+fn make_remark_proposal() -> Call {
```

Comment:
This should probably be that function, moved centrally, and then `make_proposal` can probably be removed in most cases